unit/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************\
+
+Module: Unit tests for satcheck_minisat2
+
+Author: Peter Schrammel
+
+\*******************************************************************/
+
+/// \file
+/// Unit tests for satcheck_minisat2
+
+#include <testing-utils/use_catch.h>
+
+#include <solvers/prop/literal.h>
+#include <solvers/sat/satcheck_minisat2.h>
```

Comment:
This will only work if we have enabled this solver at compile time. I'd ask for it to be wrapped in a suitable `#ifdef`.